src/main/java/cam72cam/immersiverailroading/entity/LocomotiveSteam.java

```diff
@@ -708,11 +709,23 @@ public void onUpdate() {
 			// 10% over max pressure OR
 			// Half max pressure and high boiler temperature
 			//EXPLODE
-			if (Config.ConfigDamage.explosionsEnabled) {
+			if (Config.ConfigDamage.explosionsEnabled && Config.ConfigDamage.explosionEnvDamageEnabled) {
```

Comment:
you can simplify the if statements.
if (Config.ConfigDamage.explosionsEnabled) {
if (Config.ConfigDamage.explosionEnvDamageEnabled) {

} else {

}
}